### PR TITLE
scripts: image: BUILDER_NAME in image filename

### PR DESCRIPTION
**Pull request by sky42src**
to host builds from more then one builder and create a releases.json for each builder, there should be the builder name in the filename and the releases.py needed to be extended to create LibreELEC-9.2-Milhouse just from the file name.

### CODE REVIEW

**Comment by 5schatten** [comment]
Does `releases.json` recognize builder names already?

**Comment by sky42src** [comment]
@5schatten no, but i did take a wish for it here https://github.com/LibreELEC/release-scripts/pull/3#issuecomment-528709206

**Comment by MilhouseVH** [comment]
I don't generally have a problem with this.... but it does make parsing the filename a little trickier as there's now an optional component, but there's not much we can do about that because of backward compatibility.

File: scripts/image [hunk 1]
```diff
@@ -95,13 +95,21 @@ if [ -n "${CUSTOM_VERSION}" ]; then
 fi
 
 LIBREELEC_ARCH="${DEVICE:-${PROJECT}}.${TARGET_ARCH}"
-TARGET_VERSION="${LIBREELEC_ARCH}-${LIBREELEC_VERSION}"
+if [ "${LIBREELEC_BUILD}" = "official" ]; then
```
Comment (MilhouseVH) [inline scripts/image]: I would change this test to be:
```suggestion
if [ "${OFFICIAL}" = "yes" ]; then
```
as that avoids encoding `"official"` more than once.

**Comment by sky42src** [comment]
@MilhouseVH i committed your suggestion

**Comment by MilhouseVH** [comment]
Let's see if anyone complains...

**Pull request merged**